### DISCUSSION TITLE
Type of input and output of each span visible in the UI

### DISCUSSION THREAD

**Discussion by arthurGrigo** [langfuse/langfuse#2278] [Ideas]
### Describe the feature or potential improvement

When debugging complex prompt chains built with langchain it could be really helpfull to know what type the input or output of a specific step in the chain had.

Currently, even if the step returns a json string it is shown as a nicely structured object in the langfuse UI (which is nice).

Sometimes it is neccessary to know if the output was a json string, a dict, an object or a pydantic object.

Would love to see this information in the UI.

**Comment by marcklingen** [comment]
agree, this'd be useful. currently, we opted to try to parse outputs by default to render them nicely in UI. A "real" raw mode would be helpful here. Let's keep track of overall interest for this here
